Server/src/server/gameCenter/models/game/Game.java

```diff
@@ -40,6 +42,7 @@
     private int reward;
     private boolean isFinished;
     private ArrayList<Account> observers = new ArrayList<>();
+    private Timer timer;
```

Comment:
Did you look into using [`ScheduledExecutorService`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ScheduledExecutorService.html) instead of `Timer` by chance? [This SO question](https://stackoverflow.com/questions/409932/java-timer-vs-executorservice) has some comparisons with the benefits laid out in some of the answers.

Reply:
TBH I've searched for something that would act as a Timer first: 

- OFC the SES isnt affected by system clock changes, but perhaps the sever will not be subject to system clock changes.
- the Timer object has one thread, where SES is made for handling multiple threads: we want to do a countdown for a player turn, nothing more. I'll look into that more when I have time.

Reply:
I think there's a [helper method](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Executors.html#newSingleThreadScheduledExecutor(java.util.concurrent.ThreadFactory)) that gets you to the same single threaded spot as `Timer` pretty much right away.

Reply:
Will look into that